### PR TITLE
SD-1875: Make sure titles are unique within a standard

### PR DESCRIPTION
**Pull request by HenrikHL**
[SD-1875](https://dcsa.atlassian.net/browse/SD-1875): Make sure titles are unique within a standard. The titles now reflect the object name and no titles within the same standard use the same name

[SD-1875]: https://dcsa.atlassian.net/browse/SD-1875?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by HenrikHL** [comment]
adding you @jkosternl because I was wondering why the Bot does not do a review...?

File: ebl/v3/EBL_v3.0.0.yaml [hunk 1]
```diff
@@ -5454,7 +5454,7 @@ components:
 
     PartyContactDetailWithPattern:
       type: object
-      title: Party Contact Detail
+      title: Party Contact Detail (with phone pattern)
```
Comment (jkosternl) [inline ebl/v3/EBL_v3.0.0.yaml:5457]: Does the 'phone pattern' make sense if you can also supply an email address? At least, I try to match this change with the description below.

**Pull request merged**